applications/salk-portal/src/components/ExplorerSidebarDialog.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from "react";
+import {
+  Box,
+  makeStyles,
+  Grid,
+  Typography,
+  Link,
+  Container
+} from "@material-ui/core";
+import { switchActiveColor, headerBorderColor } from "../theme";
+import LOGO from "../assets/images/logo_icon.svg"
+import Modal from "./common/BaseDialog";
+
+const useStyles = makeStyles(() => ({
+  learnMore: {
+    '& .details': {
+      flexGrow: 1,
+
+      '& .detail-block + .detail-block': {
+        marginTop: '1.5rem',
+      },
+
+      '& .logo-container': {
+
+        '& .logo': {
+          display:"flex",
+          justifyContent:"center",
+          alignItems:"center"
+        },
+
+        '& .MuiContainer-root': {
+            border: "0.063rem solid",
+            width: "5rem",
+            height: "5rem",
+            borderRadius: "50%",
+            color:headerBorderColor
+        }
+      }, 
+
+  
+      '& .MuiTypography-root': {
```

Comment:
@paularah when you separate the details class and avatar class, then these following element styling can be a part of detail. Gives clear css separation